packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -89,6 +100,11 @@ abstract class RenderToggleable extends RenderConstrainedBox implements Semantic
     markNeedsPaint();
   }
 
+  /// Called when the control finishes animating to the new value.
+  ///
+  /// The control is considered interactive (see [isInteractive]) if this
+  /// callback is non-null. For example, a non-interactive checkbox is dislayed
```

Comment:
replace second sentence with something like: `If the callback is null, then the control is disabled, and non-interactive. A disabled checkbox, for example, is displayed using a grey color and its value cannot be changed.`

(in any case, dislayed -> displayed)